shared/src/theme/index.ts

```diff
@@ -0,0 +1,20 @@
+import mustache from 'mustache'
+import fs from 'fs'
+import path from 'path'
+
+const generateThemedHTMLEmail = async (emailThemeFields: {
+  body: string
+  agencyName?: string
+  agencyLogoURI?: string
+  unsubLink: string
+}): Promise<string> => {
+  const htmlTemplate = await fs.promises.readFile(
+    path.resolve(__dirname, './email-theme.mustache'),
+    'utf-8'
+  )
```

Comment:
Actually, should I move this out as a global constant? Seems wiser to only read the file once during initial run instead of for every message.
Or alternatively, refactor this entire file as a singleton class so it will get loaded a max of 1 time but will not be loaded if it isn't used.

Reply:
Makes sense. Though slight preference for the second method. Ideally, importing a module should not result in any side effect.